src/readiness.ts

```diff
@@ -0,0 +1,39 @@
+import { Bee, Utils } from '@ethersphere/bee-js'
+import { logger } from './logger'
+import { StampsManager } from './stamps'
+
+export async function tryUploadingSingleChunk(stampsManager: StampsManager): Promise<boolean> {
+  const chunk = makeChunk()
+  const bee = new Bee(process.env.BEE_API_URL || 'http://localhost:1633')
+  try {
+    await bee.uploadChunk(stampsManager.postageStamp, chunk, { timeout: 3_000, deferred: false })
```

Comment:
Why is this `deferred: false` (or with other words direct upload)?

In the readiness check the goal is to test if the node is working correctly, not that the network is working correctly which may depend on other nodes on the network.